src/cmd/sql_cmd.h

```diff
@@ -1469,6 +1468,18 @@ void Shell() {
                 continue;
             }
         }
+        if (buffer.find_last_of(';') != std::string::npos && buffer.back() != ';') {
```

Comment:
what's the use of semicolon ? 

so you'd like to cover the case like `show databases; show ` ? I checked mysql but they treat the second `show` as unterminated.

Reply:
I think mysql execute "show databases" and buffer "show" and wait for command, mysql support multiple commands. Openmldb support single command temporarily. So I will treat "show databases; show" as an illegal command. "Show databases; show databases;" will be consider as single command and parse failed. "Show database;      " will be executed.